schemas/io-package.json

```diff
@@ -1,1275 +1,1270 @@
-              "$ref": "#/definitions/multilingual"
-            }
-          },
-          "additionalProperties": false
-        },
-        "desc": {
-          "description": "Description, object for multilingual description",
-          "$ref": "#/definitions/multilingual"
-        },
-        "messagebox": {
-          "description": "true if messagebox supported. Hence, the adapter can receive sendTo messages.",
-          "type": "boolean"
-        },
-        "readme": {
-          "description": "Url of the ReadMe file",
-          "type": "string"
-        },
-        "adminUI": {
-          "description": "Type of the admin UI",
-          "type": "object",
-          "additionalProperties": false,
-          "required": [
-            "config"
-          ],
-          "properties": {
-            "custom": {
-              "description": "UI type of custom tab inside admin UI",
-              "type": "string",
-              "const": "json"
-            },
-            "config": {
-              "description": "UI type of config page inside admin UI",
-              "type": "string",
-              "enum": [
-                "html",
-                "json",
-                "materialize",
-                "none"
-              ]
-            },
-            "tab": {
-              "description": "UI type of tab inside admin UI",
-              "type": "string",
-              "enum": [
-                "html",
-                "materialize"
-              ]
-            }
-          }
-        },
-        "docs": {
-          "description": "The structure like {\"en\": \"docs/en/README.md\", \"de\": [\"docs/de/README.md\", \"docs/de/README1.md\"]} that describes the documentation if not in README.md",
-          "type": "object",
-          "required": [
-            "en"
-          ],
-          "patternProperties": {
-            "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
-              "type": [
-                "string",
-                "array"
-              ],
-              "minItems": 1,
-              "items": {
-                "type": "string"
-              }
-            }
-          },
-          "additionalProperties": false
-        },
-        "authors": {
-          "description": "Array of authors as string",
-          "type": "array",
-          "minItems": 1,
-          "items": {
-            "anyOf": [
-              {
-                "type": "string"
-              },
-              {
-                "type": "object",
-                "additionalProperties": false,
-                "required": [
-                  "name"
-                ],
-                "properties": {
-                  "name": {
-                    "type": "string"
-                  },
-                  "email": {
+    "$schema": "https://json-schema.org/draft-07/schema",
```

Comment:
is that really also relevant for schema urls??

Reply:
Doesn't hurt IMO, so why not?

Reply:
kk, so we "just" need to fix the intents or take over manually ... for the rest